lib/bolt/config.rb

```diff
@@ -50,6 +50,9 @@ class Config
                                     "OS-specific PATH separator.",
       "plugin_hooks"             => "Which plugins a specific hook should use.",
       "plugins"                  => "A map of plugins and their configuration data.",
+      "project_name"             => "The namespace for project artifacts. Setting this also opts the user into "\
```

Comment:
Any chance the option could be `project-name` instead? Most of our config options use hyphens and I think it would be nice to maintain consistency as much as we can.

Reply:
We haven't been very consistent about it, but we tend to use underscores for config that can't be set with CLI flags (`apply_settings`, `plugin_hooks`, etc.) and hyphens for config that can be set on the CLI. I think the question is more whether we expect project-name to ever be configurable on the CLI.